Examples/WildeGuess/WildeGuess/Components/InteractiveQuoteComponent.mm

```diff
@@ -11,12 +11,8 @@
 
 #import "InteractiveQuoteComponent.h"
 
-#import <ComponentKit/CKComponentGestureActions.h>
-#import <ComponentKit/CKComponentScope.h>
+#import <ComponentKit/ComponentKit.h>
 #import <ComponentKit/CKComponentSubclass.h>
-#import <ComponentKit/CKCompositeComponent.h>
-#import <ComponentKit/CKOverlayLayoutComponent.h>
-#import <ComponentKit/CKStackLayoutComponent.h>
```

Comment:
I kind of like using the explicit imports in the example project, feels more explanatory